assets/js/index.js

```diff
@@ -13,24 +13,32 @@ class App {
     this.appContainer = container;
     this.mobileNavIcon = this.appContainer.querySelectorAll(App.selectors.dataMobileNavIcon);
     this.mobileNav = this.appContainer.querySelector(App.selectors.dataMobileMenu);
+    this.page = this.appContainer.querySelectorAll(App.selectors.dataPage);
   }
 
   _bindEvents() {
+    this.page.forEach((element) => {
+      element.addEventListener('click', () => this.closeMobileMenu())
+    })
+
     this.mobileNavIcon.forEach(icon => {
-      icon.addEventListener('click', () => {
-        this.toggleMobileMenu();
-      })
+      icon.addEventListener('click', () => this.toggleMobileMenu())
```

Comment:
We'd never be able to remove this event listener since we're passing in an anonymous function.

Since `this.toggleMobileMenu` and `this.closeMobileMenu` are already named functions and have a symbolized pointer in memory – it's best to just pass those functions in to listeners:

```js
element.addEventListener('click', this.closeMobileMenu);
icon.addEventListener('click', this.toggleMobileMenu);
```

Reply:
I also believe we should be only listening for a page click when the `mobileMenu` is opened, and then the listener is remove when the `mobileMenu` is closed. Otherwise this function will run each time someone clicks on desktop – even if they never see the menu.